homeassistant/components/media_extractor/__init__.py

```diff
@@ -181,3 +180,19 @@ def get_stream_query_for_entity(self, entity_id: str | None) -> str:
             )
 
         return default_stream_query
+
+
+def get_best_stream(formats: list[dict[str, Any]]) -> str:
+    """Return the best quality stream.
+
+    As per
+    https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/common.py#L128.
+    """
+
+    return cast(str, formats[len(formats) - 1]["url"])
+
+
+def get_best_stream_youtube(formats: list[dict[str, Any]]) -> str:
+    """YouTube requests also include manifest files. Skip them."""
```

Comment:
Please update the description to explain how this skips manifest files.
Do manifest files not have a filesize or something?